pulseapi/profiles/apps.py

```diff
@@ -1,5 +1,10 @@
 from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
```

Comment:
oh god no let's never do this O_O

it has a name. Just use that name. Obfuscating literally makes our own dev work harder.